Add `osu! Austria @ Conventions 2024` news post (#11802)
ppy/osu-wiki: news/2024/2024-07-23-osu-austria-conventions-2024.md

Join us at the biggest Anime and Manga convention in Austria, [AniNite](https://www.aninite.at/en/), which has over 25,000 yearly visitors. Meet our community from **August 9th to 11th, 2024** at our osu! booth. In cooperation with [Friendly Fire](https://friendlyfireesports.com/), we have organized 12 PCs for you to play on and show everyone what you're made of.

During this time, a **battle royale-styled** osu! tournament will be hosted. Make sure to register during the event and take home the trophy!

## YuniCon 2024

![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/yunicon.png)

Next up, we're setting up an osu! booth at [Yunicon 2024](https://www.yunicon.at/en/). During all 4 days of the convention, you'll be able to meet our community, ask questions and play on 10 pre-organized computers provided by [Streamhouse](https://streamhouse.at/)!

On top of that, you'll also be able to witness the biggest osu! 1v1 LAN tournament Austria has ever seen! The top players of the tournament will ferociously duke it out for some handsome physical prizes and a bespoke profile badge on the main stage of the event.

> Top 16 players won't go home empty-handed as they will receive some nice goodies as well!

Reviewer: this doesnt have to be a new sentence, it could be tacked onto the existing one

Author: did something here

---
layout: post
title: osu! Austria @ Conventions 2024
date: 2024-07-23 12:00:00 +0000
---

The osu! Austria community proudly presents its appearance at not one, but two upcoming conventions in 2024!

![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/Banner.png)

The Austrian osu! community is excited to announce their presence at two upcoming conventions. Next to osu! booths, there will also be LAN tournaments, in which you can show off your skills and win some nice goodies. So what are you waiting for? Get all the information you need down below!

## AniNite 2024

![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/aninite.png)

> Join us at the biggest Anime and Manga convention in Austria, [AniNite](https://www.aninite.at/en/), which has over 25,000 yearly visitors. Meet our community from **August 9th to 11th, 2024** at our osu! booth. In cooperation with [Friendly Fire](https://friendlyfireesports.com/), we have organized 12 PCs for you to play on and show everyone what you're made of.

Reviewer 2: ```suggestion
Join us at the biggest Anime and Manga convention in Austria, [AniNite](https://www.aninite.at/en/), which has over 25,000 yearly visitors. Meet our community from **August 9th to 11th, 2024** at our osu! booth. In cooperation with [Friendly Fire](https://friendlyfireesports.com/), we have organised 12 PCs for you to play on and show everyone what you're made of.
```

> During this time, we will host a tournament, **Battle Royale** style. Register live on the spot and take home the trophy!

Reviewer 3: ```suggestion
During this time, a **battle royale-styled** osu! tournament will be hosted. Make sure to register during the event and take home the trophy!
```

> During this time, a **battle royale-styled** osu! tournament will be hosted. Make sure to register during the event and take home the trophy!

Reviewer 4: en dash when it connects terms with spaces between them

```suggestion
During this time, we'll host a **battle royale–styled** osu! tournament. Make sure to register during the event and take home the trophy!
```

## YuniCon 2024

![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/yunicon.png)

> Next up, we're setting up an osu! booth at [Yunicon 2024](https://www.yunicon.at/en/).

Reviewer 3: ```suggestion
Next up, we're setting up an osu! booth at [Yunicon 2024](https://www.yunicon.at/en/). During all 4 days of the convention, you'll be able to meet our community, ask questions and play on 10 pre-organized computers provided by [Streamhouse](https://streamhouse.at/)!
```

> On all 4 days of the convention, you'll be able to meet our community, ask questions and play on 10 pre-organized computers.

Reviewer 3: ```suggestion
```

> Computers are being provided by [Streamhouse](https://streamhouse.at/).

Reviewer 3: ```suggestion
```

> Next up, we're setting up an osu! booth at [Yunicon 2024](https://www.yunicon.at/en/). During all 4 days of the convention, you'll be able to meet our community, ask questions and play on 10 pre-organized computers provided by [Streamhouse](https://streamhouse.at/)!

Reviewer 2: ```suggestion
Next up, we're setting up an osu! booth at [Yunicon 2024](https://www.yunicon.at/en/). During all 4 days of the convention, you'll be able to meet our community, ask questions and play on 10 pre-organised computers provided by [Streamhouse](https://streamhouse.at/)!
```

> Next up, we're setting up an osu! booth at [Yunicon 2024](https://www.yunicon.at/en/). During all 4 days of the convention, you'll be able to meet our community, ask questions and play on 10 pre-organised computers provided by [Streamhouse](https://streamhouse.at/)!

Reviewer 4: ```suggestion
Next up, we're setting up an osu! booth at [Yunicon 2024](https://www.yunicon.at/en/). During all 4 days of the convention, you'll be able to meet our community, ask questions and play on 10 pre-organised computers!
```

> On top of that, there will also be the biggest osu! 1v1 LAN tournament Austria has ever seen! The top 2 players will fight it out for real life prizes and a profile badge on the main stage of the event.

Reviewer 3: ```suggestion
On top of that, you'll also be able to witness the biggest osu! 1v1 LAN tournament Austria has ever seen! The top players of the tournament will ferociously duke it out for some handsome physical prizes and a bespoke profile badge on the main stage of the event.
```

Top 16 players won't go home empty-handed as they will receive some nice goodies as well!

Join our [osu! Austrian Tournaments discord server](https://discord.gg/vRbC3bw) to not miss out on any updates regarding this tournament.

### Tournament schedule

| Event | Timestamp (UTC+1) |
| --: | :-- |

> | Registrations | 2024-10-31 (13:00-23:00) |

Author: did

> On top of that, you'll also be able to witness the biggest osu! single elimination bracket based 1v1 LAN tournament Austria has ever seen! 
> You'll be able to sign-up for the tournament online. Please join [our Discord server](https://discord.gg/dNFuHNmZF4), to get notified when sign-ups open.
> Qualifiers will be played out online as well, scheduling your lobby will also be handled over at our **Discord server**. After you qualified for the round of 16, **you're required to show up**. A no-show will automatically **eliminate** you from the tournament and open your spot for the next possible seed that is present at the event. Bring the peripherals you're comfortable with and prepare to bash your opponents! 

Reviewer 4: ```suggestion
On top of that, you'll also be able to witness the biggest osu! 1v1 LAN tournament with a single-elimination bracket that Austria has ever seen!

You'll be able to sign-up for the tournament online. Please join our [Discord server](https://discord.com/invite/dNFuHNmZF4) to get notified when sign-ups open.

Qualifiers will be played out online as well and scheduling your lobby will be handled over at our *Discord server*. After you qualified for the Round of 16, **you're required to show up**. A no-show will automatically **eliminate** you from the tournament and open your spot for the next possible seed that is present at the event. Bring the peripherals you're comfortable with and prepare to bash your opponents!
```

> The top 2 players of the tournament will ferociously duke it out for some handsome physical prizes and a bespoke profile badge on the main stage of the event and top 16 players won't go home empty-handed as they will receive some nice goodies as well!

Reviewer 4: ```suggestion
The top 2 players of the tournament will ferociously duke it out for some handsome physical prizes and a bespoke profile badge on the main stage of the event, and the top 16 players won't go home empty-handed either as they will receive some nice goodies as well!
```

| Qualifiers | 2024-11-01 (10:00-19:00) |
| Round of 16 | 2024-11-02 (10:00-14:00) |
| Quarterfinals | 2024-11-02 (14:30-17:00) |
| Semifinals | 2024-11-02 (17:30-20:00) |
| Finals | 2024-11-03 (10:00-13:00) |
| Grandfinals | 2024-11-03 (13:15) |

---

For all the latest news on the happenings throughout the event, join the [osu! Austria meetups & events Discord server](https://discord.gg/dNFuHNmZF4) or follow [@osuaustria on Twitter](https://twitter.com/osuaustria)

> We can’t wait to see you there!

Reviewer 3: ```suggestion
We can't wait to see you there!
```

> | Registrations | 2024-10-31 (13:00–23:00) |
> | Qualifiers | 2024-11-01 (10:00–19:00) |
> | Round of 16 | 2024-11-02 (10:00–14:00) |
> | Quarterfinals | 2024-11-02 (14:30–17:00) |
> | Semifinals | 2024-11-02 (17:30–20:00) |
> | Finals | 2024-11-03 (10:00–13:00) |
> | Grandfinals | 2024-11-03 (13:15) |

Reviewer 4: for news posts we usually write dates out for readability

```suggestion
| Registrations | 31 October 2024 (13:00–23:00) |
| Qualifiers | 1 November 2024 (10:00–19:00) |
| Round of 16 | 2 November 2024 (10:00–14:00) |
| Quarterfinals | 2 November 2024 (14:30–17:00) |
| Semifinals | 2 November 2024 (17:30–20:00) |
| Finals | 3 November 2024 (10:00–13:00) |
| Grand Finals | 3 November 2024 (13:15) |
```

> For all the latest news on the happenings throughout the event, join the [osu! Austria meetups & events Discord server](https://discord.gg/dNFuHNmZF4) and follow [@osuaustria on Twitter](https://twitter.com/osuaustria)

Reviewer 4: ```suggestion
For all the latest news on the happenings throughout the event, join the [osu! Austria meetups & events Discord server](https://discord.com/invite/dNFuHNmZF4) and follow [@osuaustria on Twitter](https://twitter.com/osuaustria)
```

Next to osu! booths, there will also be LAN tournaments, in which you can show off your skills and win some nice goodies. What are you waiting for? Get all the information you need down below!

## AniNite 2024

![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/aninite.png)

Join us at the biggest anime and manga convention in Austria, [AniNite](https://www.aninite.at/en/), which has over 25,000 yearly visitors. You can find us from **August 9th to 11th** at our osu! booth, where there will be 12 PCs for you to play on and show everyone what you're made of.

During this time, we'll host a **battle royale–styled** osu! tournament. Make sure to register during the event and take home the trophy!

## YuniCon 2024

![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/yunicon.png)

Next up, we're setting up an osu! booth at [Yunicon 2024](https://www.yunicon.at/en/). During all 4 days of the convention, you'll be able to meet our community, ask questions and play on 10 pre-organised computers!

On top of that, you'll also be able to witness the biggest osu! 1v1 LAN tournament with a single-elimination bracket that Austria has ever seen!

> You'll be able to sign-up for the tournament online. Please join our [Discord server](https://discord.com/invite/dNFuHNmZF4) to get notified when sign-ups open.

Reviewer: ```suggestion
You'll be able to sign up for the tournament online. Please join our [Discord server](https://discord.com/invite/dNFuHNmZF4) to get notified when sign-ups open.
```

Qualifiers will be played out online as well and scheduling your lobby will be handled over at our *Discord server*. After you qualified for the Round of 16, **you're required to show up**. A no-show will automatically **eliminate** you from the tournament and open your spot for the next possible seed that is present at the event. Bring the peripherals you're comfortable with and prepare to bash your opponents!

> The top 2 players of the tournament will ferociously duke it out for some handsome physical prizes and a bespoke profile badge on the main stage of the event, and the top 16 players won't go home empty-handed either as they will receive some nice goodies as well!

Reviewer: I assume the physical prizes will only go to the winner? rewrote this the way i liked it more, but its subjective ofc
```suggestion
The top 2 players of the tournament will ferociously duke it out on the main stage of the event, with the winner receiving handsome physical prizes and a bespoke profile badge. The top 16 players won't go home empty-handed either, as they will receive some nice goodies as well!
```

### Tournament schedule

| Event | Timestamp (UTC+1) |
| --: | :-- |
| Registrations | 31 October 2024 (13:00–23:00) |
| Qualifiers | 1 November 2024 (10:00–19:00) |
| Round of 16 | 2 November 2024 (10:00–14:00) |
| Quarterfinals | 2 November 2024 (14:30–17:00) |
| Semifinals | 2 November 2024 (17:30–20:00) |
| Finals | 3 November 2024 (10:00–13:00) |
| Grand Finals | 3 November 2024 (13:15) |

---

> For all the latest news on the happenings throughout the event, join the [osu! Austria meetups & events Discord server](https://discord.com/invite/dNFuHNmZF4) and follow [@osuaustria on Twitter](https://twitter.com/osuaustria)

Reviewer: ```suggestion
For all the latest news on the happenings throughout the event, join the [osu! Austria meetups & events Discord server](https://discord.com/invite/dNFuHNmZF4) and follow [@osuaustria on Twitter](https://twitter.com/osuaustria).
```

## AniNite 2024

![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/aninite.png)

Join us at the biggest anime and manga convention in Austria with over 25,000 visitors each year: **[AniNite](https://www.aninite.at/en/)**! Located near the beautiful Danube River at Austria Center in Vienna, you can find us from **August 9th to 11th** at our osu! booth, where there will be 12 PCs for you to play on and show everyone what you're made of.

During this time, we'll host a **battle royale–styled** osu! tournament. Make sure to register during the event and take home the trophy!

## YuniCon 2024

![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/yunicon.png)

Next up, we're setting up an osu! booth at **[Yunicon 2024](https://www.yunicon.at/en/)**, situated at Mutliversum Schwechat. During all 4 days of the convention, you'll be able to meet our community, ask questions and play on 10 pre-organised computers!

On top of that, you'll also be able to witness the biggest osu! 1v1 LAN tournament with a single-elimination bracket that Austria has ever seen!

![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/yunicupbannerfinals.png)

> You'll be able to sign up for the tournament online. Please join our [Discord server](https://discord.com/invite/dNFuHNmZF4) to get notified when sign-ups open.

Reviewer: In relation to [this](https://github.com/ppy/osu-wiki/pull/11802/files#r1688447290):
```suggestion
You'll be able to sign up for the tournament online. Please join our [Discord server](https://discord.com/invite/dNFuHNmZF4) where you can get notified when sign-ups open, and where organisation of the Qualifiers will happen.
```

> Qualifiers will be played out online as well and scheduling your lobby will be handled over at our *Discord server*. After you qualified for the Round of 16, **you're required to show up**. A no-show will automatically **eliminate** you from the tournament and open your spot for the next possible seed that is present at the event. Bring the peripherals you're comfortable with and prepare to bash your opponents!

Reviewer: `Qualifiers will be played out online as well and scheduling your lobby will be handled over at our *Discord server*`
not sure if this needs to be italicised? i'd either remove the italicising or that sentence altogether and merge it with the one above where the discord server is actually linked

Author: ermm i dont even remember why i italicized that. the reason why i'd want to keep that, is to make sure people know how it works with the scheduling. rather one question less here. but ill remove the italicization.

Reviewer: check out what i just suggested

Author: applied all

> Qualifiers will be played out online as well and scheduling your lobby will be handled over at our *Discord server*. After you qualified for the Round of 16, **you're required to show up**. A no-show will automatically **eliminate** you from the tournament and open your spot for the next possible seed that is present at the event. Bring the peripherals you're comfortable with and prepare to bash your opponents!

Reviewer: ```suggestion
After you qualify for the Round of 16, **you're required to show up**. A no-show will automatically **eliminate** you from the tournament and open your spot for the next possible seed that is present at the event. Bring the peripherals you're comfortable with and prepare to bash your opponents!
```

![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/aninite.png)

> Join us at the biggest anime and manga convention in Austria with over 25,000 visitors each year: **[AniNite](https://www.aninite.at/en/)**! Located near the beautiful danube at Austria Center in Vienna. You can find us from **August 9th to 11th** at our osu! booth, where there will be 12 PCs for you to play on and show everyone what you're made of.

Reviewer: ```suggestion
Join us at the biggest anime and manga convention in Austria with over 25,000 visitors each year: **[AniNite](https://www.aninite.at/en/)**! Located near the beautiful Danube River at Austria Center in Vienna. You can find us from **August 9th to 11th** at our osu! booth, where there will be 12 PCs for you to play on and show everyone what you're made of.
```
